go/private/actions/compile.bzl

```diff
@@ -103,5 +103,7 @@ def _bootstrap_compile(go, sources, out_lib, gc_goopts):
         inputs = sources + go.sdk_files + go.sdk_tools,
         outputs = [out_lib],
         mnemonic = "GoCompile",
-        command = "export GOROOT=$(pwd)/{} && export GOROOT_FINAL=GOROOT && {} {}".format(go.root, go.go.path, " ".join(args)),
+        # workaround: go link tool needs some features of gcc to complete the job on Arm platform.
+        # So, PATH for 'gcc' is required here on Arm platform.
```

Comment:
Why is gcc needed for compiling Go code? My expectation is that it should only be required for linking.